python/lsst/sims/maf/slicers/baseSpatialSlicer.py

```diff
@@ -204,35 +204,15 @@ def _presliceFootprint(self, simData):
         if self.verbose:
             "Created lookup table after checking for chip gaps."
 
-    def _treexyz(self, ra, dec):
-        """Calculate x/y/z values for ra/dec points, ra/dec in radians."""
-        # Note ra/dec can be arrays.
-        x = np.cos(dec) * np.cos(ra)
-        y = np.cos(dec) * np.sin(ra)
-        z = np.sin(dec)
-        return x, y, z
-
     def _buildTree(self, simDataRa, simDataDec, leafsize=100):
-        """Build KD tree on simDataRA/Dec and set radius (via setRad) for matching.
+        """Build KD tree on simDataRA/Dec using utility function from mafUtils.
 
         simDataRA, simDataDec = RA and Dec values (in radians).
         leafsize = the number of Ra/Dec pointings in each leaf node."""
-        if np.any(np.abs(simDataRa) > np.pi*2.0) or np.any(np.abs(simDataDec) > np.pi*2.0):
-            raise ValueError('Expecting RA and Dec values to be in radians.')
-        x, y, z = self._treexyz(simDataRa, simDataDec)
-        data = list(zip(x, y, z))
-        if np.size(data) > 0:
-            try:
-                self.opsimtree = kdtree(data, leafsize=leafsize, balanced_tree=False, compact_nodes=False)
-            except TypeError:
-                self.opsimtree = kdtree(data, leafsize=leafsize)
-        else:
-            raise ValueError('SimDataRA and Dec should have length greater than 0.')
+        self.opsimtree = _buildTree(simDataRa,
+                                    simDataDec,
+                                    leafsize)
```

Comment:
I acknowledge that this is super nitpicky. 

However, this becomes kind of hard to read because it looks like "_buildTree calls _buildTree?". 
To save us brain confusion when we forget about these changes and why they happened, could you please change the import and calls, just in this file? 
e.g.
```
import lsst.sims.utils as simsUtils
sx, sy, sz = simsUtils._xyz_from_ra_dec
self.opsimtree = simsUtils._buildTree(simDataRa, simDataDec, leafsize)
```

Reply:
Ok... done... I'm doing a build on Jenkins just to make sure and will merge it afterwards...